policyengine_us/variables/gov/states/az/hhs/tanf/az_hhs_tanf_earned_income.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class az_tanf_earned_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Earned income for the Arizona Cash Assistance"
+    definition_period = YEAR
```

Comment:
@PavelMakarchuk let's make all such programs monthly from here on
```suggestion
    definition_period = MONTH
```